bindings/js/.npmignore

```diff
@@ -0,0 +1,7 @@
+*.wast
+doc/
+main.html
+main.js
+browser.js
+README
+Hacl_Curve25519_64_Slow.wasm
```

Comment:
remove this one too?

Reply:
Hum no we need this one. Indeed in order to publish a new version of the package I `make` for Kremlin to generate the wasm files in `/dist/wasm`, and then files are copied from `bindings/js` to `dist/wasm`. At this point, `dist/wasm/.npmignore` effectively controls what goes into the npm package. But `Hacl_Curve25519_64_Slow.wasm` is generated by Kremlin and is still in `dist/wasm`, so we have to explicitly ignore it in `.npmignore`. Indeed, the behaviour of `.npmignore` is that is ignores whatever is written in `.gitignore` (sic)...

Reply:
aah, ok, thanks for the explanation!

Reply:
But you can change the Makefile target so that Kremlin won't generate `Hacl_Curve25519_64_Slow.wasm`

Reply:
Thanks @polubelova, I added a `bundle` option that should do that :)